package.json

```diff
@@ -18,6 +18,7 @@
   "license": "ISC",
   "private": false,
   "dependencies": {
+    "damerau-levenshtein": "^1.0.4",
```

Comment:
Let's peg the exact version so we don't unknowingly upgrade versions in the future. Removing the `^` and re-running `yarn` will do the trick.

Reply:
Thanks -- I'll follow up with this in my documentation PR.